src/Agent/NewRelic/Agent/Core/WireModels/MetricWireModel.cs

```diff
@@ -16,7 +15,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net;
-using InternalMetricName = NewRelic.Agent.Core.Metric.MetricName;
+using InternalMetricName = NewRelic.Agent.Core.Metrics.MetricName;
```

Comment:
Should we get a story added to the backlog to address this naming conflict that we are currently working around?

Reply:
Yeah, I'll create one.

Reply:
https://github.com/newrelic/newrelic-dotnet-agent/issues/1900